app/models/about.rb

```diff
@@ -82,7 +98,26 @@ def stats
       likes_last_day: UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 1.days.ago).count,
       likes_7_days: UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 7.days.ago).count,
       likes_30_days: UserAction.where(action_type: UserAction::LIKE).where("created_at > ?", 30.days.ago).count
-    }
+    }.merge(plugin_stats: plugin_stats)
```

Comment:
From an API perspective, I'm not sure if having plugin stats under an additional `plugin_stats` key is a user concern. Also, it makes it harder to process the stats because you can't just loop through the key and values anymore. I feel like if we're going to add stuff into this hash, it should follow the same structure

```
...
chat_messages_last_day
chat_messages_7_days
...

I wonder if we can make use of the `hidden_plugin_stat_groups` but flip it around to be `display_plugin_stat_group`. Then on the client side we can just loop and add the stats by using a convention. `stats["#{pluginStatGroupName}_7_days"]` for example.
```

Reply:
Oh yeah that's a better way of thinking about it, then the `display_plugin_stat_group` array would always have the core ones and then certain plugin ones if the plugin says they should be shown in the UI. And yeah would be nicer to not have `plugin_stats` as a sub-key :+1: